pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
+  intl: ^0.19.0
```

Comment:
currently we are accept only extension that are not depends on any other library

Reply:
Other all are looks good, currently can you make a pr that not depend on any other package

Reply:
I understand that you don't want to have dependencies, but sometimes that is what can make the use of extensions (such as throwing phone numbers or emails) more meaningful.